File: spring-kafka-producer/spring-webflux-2/src/main/java/windoctor7/github/io/spring5/reactive/springwebflux2/Student.java
package windoctor7.github.io.spring5.reactive.springwebflux2;

import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

/**
 * Created by Ascari Q. Romo Pedraza - molder.itp@gmail.com on 19/10/2017.
 */
@Document(collection = "students")
public class Student {

    private String id;
    private String name;
    private List<Score> scores;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Score> getScores() {
        return scores;
    }

    public void setScores(List<Score> scores) {
        this.scores = scores;
    }
}

File: spring-kafka-producer/spring-kotlin/src/main/kotlin/windoctor7/github/io/kotlin/spring/springkotlin/Student.kt
package windoctor7.github.io.kotlin.spring.springkotlin

import org.springframework.data.mongodb.core.mapping.Document

/**
 * Creado por  Ascari Q. Romo Pedraza - molder.itp@gmail.com on 21/10/17.
 */
@Document(collection = "students")
data class Student(val id:String, val name:String) {
}